package.json

```diff
@@ -27,15 +27,19 @@
     "/npm-shrinkwrap.json"
   ],
   "dependencies": {
+    "@architect-io/arc-oci": "0.1.0",
```

Comment:
Is there a reason these need to be included in the package.json? Do these not automatically land in our dnt-created package even though they're from npm/part of our import map?

Reply:
Actually we no longer use any external version of this. I ported it into our repo and Denofied it.